### PR TITLE
Supports FORCE_REBUILD.

### LINKED ISSUE #98: [makefile] Ability to force rebuild
It would be great if there is a way to instruct docker to force rebuild the image.

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #98

### CODE REVIEW

File: common.mk [hunk 1]
```diff
@@ -4,4 +4,7 @@ ifeq ($(origin DOCKER_IMAGE_NAME), undefined)
 	GCLOUD_PROJECT:=$(shell gcloud config list project --format="value(core.project)")
 	DOCKER_IMAGE_NAME:=gcr.io/$(GCLOUD_PROJECT)/google-python-compat
 endif
+ifeq ($(origin FORCE_REBUILD), undefined)
```
Comment (jonparrott) [inline common.mk]: Is this needed? What happens if it's left out?

Comment (tmatsuo) [inline common.mk]: Yeah unfortunately it errors:

```
docker build --no-cache= --pull= \
                -t gcr.io/tmatsuo-pubsub-sample/google-python-compat .
invalid boolean value "" for  --no-cache: strconv.ParseBool: parsing "": invalid syntax
See 'docker build --help'.
make: *** [build] Error 2
```

Comment (jonparrott) [inline common.mk]: Cool, just making sure.

**Comment by jonparrott** [comment]
LGTM.

**Comment by tmatsuo** [comment]
Thx

**Pull request merged**